.github/workflows/terraform.yml

```
@@ -0,0 +1,136 @@
+            } else {
+              // Create new comment
+              await github.rest.issues.createComment({
+                owner,
+                repo,
+                issue_number,
+                body: commentBody,
+              });
+              console.log('Created new comment');
+            }
+        env:
+          SUMMARY: ${{ steps.summarize.outputs.SUMMARY }}
```

actionlint (1.7.7)

property "summarize" is not defined in object type {plan: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)